src/features/game/lib/goblinMachine.ts

```diff
@@ -91,6 +92,19 @@ const makeLimitedItemsById = (items: LimitedItemRecipeWithMintedAt[]) => {
 };
 
 export function startGoblinVillage(authContext: AuthContext) {
+  // Temp check for determining if in visiting i.e. readonly state
+  const isVisiting = () => {
```

Comment:
In our application state, a user should never be visiting while the goblin machine is running.

I would suggest removing this logic from the machine as it should be handled elsewhere. We want to encapsulate only the state logic that should apply to goblin village 😃

Reply:
Okay. Reverted..